src/app/components/pricing.tsx

```
@@ -1,8 +1,36 @@
+} from '@/components/ui/dialog';
+import { Input } from '@/components/ui/input';
+import { Label } from '@/components/ui/label';
+import useMercadoPago from '@/hooks/useMercadoPago';
 
 export default function Pricing() {
+  const [showEmailModal, setShowEmailModal] = useState(false);
+  const [email, setEmail] = useState('');
+  const { createMercadoPagoCheckout } = useMercadoPago();
+
+  const handlePurchase = () => {
+    if (email && email.includes('@')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)